inference-engine/src/mkldnn_plugin/nodes/topk.cpp

```diff
@@ -77,33 +82,27 @@ class TopKImpl: public ExtLayerBase {
 
             for (size_t i = 0; i < axis; i++) {
                 axis_step *= src_dims[i];
-                if (src_data_dims[i] != dst_dims[i])
-                    THROW_IE_EXCEPTION << layer->name << " Input/output tensor dimension mismatch";
             }
             axis_dim = src_dims[axis];
             for (size_t i = (axis + 1); i < src_dims.size(); i++) {
                 axis_stride *= src_dims[i];
-                if (src_data_dims[i] != dst_dims[i])
-                    THROW_IE_EXCEPTION << layer->name << " Input/output tensor dimension mismatch";
             }
             dim = static_cast<int>(src_dims[axis]);
             before_num = count(src_dims, 0, axis);
 
-            if (layer->outData.size() == 1) {
-                addConfig(layer, { DataConfigurator(ConfLayout::PLN, Precision::FP32), DataConfigurator(ConfLayout::PLN, Precision::I32) },
-                    { DataConfigurator(ConfLayout::PLN) });
+            if (topK1Op->get_output_size() == 1) {
+                addConfig(op, {{TensorDescCreatorTypes::ncsp, Precision::FP32},
+                               {TensorDescCreatorTypes::ncsp, Precision::I32}},
+                              {{TensorDescCreatorTypes::ncsp}});
```

Comment:
We have to explicitly specify FP32 as output precision as the only precision supported by the implementation

Reply:
fixed